modules/network-observability-architecture.adoc

```diff
@@ -0,0 +1,17 @@
+//Module included in the following assemblies:
+//
+// network_observability/understanding-network-observability.adoc
+
+:_content-type: CONCEPT
+[id="network-observability-architecture_{context}"]
+= Network Observablity Operator architecture
+
+image::network-observability-architecture.png[Network Observability eBPF export architecture]
```

Comment:
Should the graphic come after the first paragraph so that folks know what is in the graphic?

Reply:
I was wondering about this too. Since you mentioned it, I think I'll move it after the text. The 2nd graphic is after the text anyways.